src/styles/Constants.ts

```diff
@@ -0,0 +1,36 @@
+export enum Colours {
+    Primary = "#05162F",
+    Secondary = "#2E5584",
+    SecondaryHover = "#284D7A",
+    Danger = "#B12323",
+    DangerHover = "#DD2B2B",
+    Black = "#000000",
+    SecondaryGray = "#676767",
+    White = "#FFFFFF",
+    Blue = "#EEF6FF",
+    BackgroundGray = "#F4F4F4",
+    BorderLightGray = "#C4C4C4",
+    DisabledBackground = "#C9C9C9",
+    DisabledText = "#5C5C5C",
+    ActiveGreen = "#43B00F",
+    InactiveGrey = "#888888",
+    TriageRed = "#FF5858",
+    TriageSelectedRed = "#FFE4E4",
+    TriageGreen = "#60CD39",
+    TriageSelectedGreen = "#D7F3CD",
+    TriageYellow = "#FFC90A",
+    TriageSelectedYellow = "#FFF2C2",
+    TriageWhite = "#CECECE",
+    TriageSelectedWhite = "#F3F3F3",
+    TriageBlack = "#000000",
+    TriageSelectedBlack = "#BFBFBF",
+    SnackbarButtonBlue = "#28B2FF",
+    Marker = "#EE502E",
+    MarkerNotClicked = "#F7B8AB",
+    CurrentLocationMarker = "#2F80ED",
```

Comment:
All of these colours are the old colours from paramedics; I assume it would be necessary to keep some such as black and white, but I'm not quite sure

Reply:
Yup you're right. If you go to the Figma that I linked in the ticket, I put a page with all the colours. It's just a matter of copy-pasting the color values and updating the name. 

https://www.figma.com/file/Le1R4aRtLWvM4Seu4DWKbE/DCC-Design-System?node-id=1%3A13

If you see in this image, on the right hand bar, you can see the name of the color when you select the color (in this case, Main Red 1). Use the to update the file. Should take you no more than 30 minutes
![image](https://user-images.githubusercontent.com/21224282/124199220-b104ea80-da8f-11eb-8293-23d50389107f.png)